packages/ses/src/lockdown-shim.js

```diff
@@ -166,36 +164,21 @@ export const repairIntrinsics = (options = {}) => {
     d`lockdown(): non supported option ${q(extraOptionsNames)}`,
   );
 
-  // Asserts for multiple invocation of lockdown().
-  if (firstOptions) {
-    for (const name of keys(firstOptions)) {
-      assert(
-        options[name] === firstOptions[name],
-        d`lockdown(): cannot re-invoke with different option ${q(name)}`,
-      );
-    }
-    return alreadyHardenedIntrinsics;
+  if (priorLockdown !== null) {
+    const error = new TypeError(
+      `Already lockded down (SES_ALREADY_LOCKED_DOWN)`,
+    );
+    // eslint-disable-next-line @endo/no-polymorphic-call
+    assert.note(error, assert.details`Prior call ${priorLockdown}`);
+    throw error;
   }
```

Comment:
Suggested simplification. Use only the parts, if any, that you agree are an improvement

```suggestion
  assert(
    priorLockdown === null,
    X`Already locked down at ${priorLockdown} (SES_ALREADY_LOCKED_DOWN)`,
    TypeError,
  );
```
assuming an earlier
```js
const { details: X } = assert;
```

Reply:
Btw, I really like this technique of capturing the earlier non-erroneous stack trace in an error in case of a later conflict.